includes/xf/opportunities_extended.js

```diff
@@ -0,0 +1,54 @@
+// TODO: opportunity.stage_names should be fetched from package config
+
+module.exports = (params) => {
+    return publish("salesforce_opportunities_extended_xf", {
```

Comment:
what's the _xf mean?

Reply:
I'm trying to denote between a `xf` or intermediate transformation table, or a `base` or `reporting` table. Probably can leave out the suffix.